schemas/com.snowplowanalytics.snowplow/web_vitals/jsonschema/1-0-0

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a web vitals tracking event. For more information on web vitals you can visit https://web.dev/vitals/.",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "web_vitals",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "cls": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": -2147483647,
+            "description": "A unitless metric for measuring visual stability because it helps quantify how often users experience unexpected layout shifts. For more information https://web.dev/cls/."
+        },
+        "fid": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": -2147483647,
+            "description": "A metric for measuring load responsiveness because it quantifies the experience users feel when trying to interact with unresponsive pages. Measured in milliseconds. For more information https://web.dev/fid/."
+        },
+        "lcp": {
+            "type": [ "number", "null" ],
+            "maximum": 2147483647,
+            "minimum": -2147483647,
```

Comment:
Should we keep the minimum for these properties below 0? As I understand they measure the speed of the page load, so negative numbers wouldn't make sense?

Reply:
There are some cases where properties like this might get reported as less than 0. This is to keep it on the safe side.

Reply:
Thinking about this. SInce this is a specific event and will not mess with any other events being sent from the browers, I added a minimum of 0. If some values come as negative it is a browser bug and should not be included in the calculations.